profile/encode.go

```diff
@@ -252,6 +253,13 @@ func (p *Profile) postDecode() error {
 		} else {
 			mappings[m.ID] = m
 		}
+
+		// If this a main linux kernel mapping with a relocation symbol suffix
```

Comment:
Maybe state explicitly that we understand it's a hack.

Reply:
Done.